build.xml

```diff
@@ -87,6 +91,36 @@
         </artifact:mvn>
     </target>
 
+    <target name="compile-tests" depends="build" description="Compile test files">
+        <path id="test.classpath">
+            <path refid="project.class.path" />
+            <fileset dir="lib" includes="**/*.jar" />
```

Comment:
Since the Android JAR just contains stubs and the robolectric classes should take precedence, I think the fileset should come before the path. Same thing below in the test target.